protocols/identify/src/lib.rs

```diff
@@ -35,20 +35,35 @@
 //!
 //! # Usage
 //!
-//! The [`Identify`] struct implements a `NetworkBehaviour` that negotiates
+//! The [`Behaviour`] struct implements a `NetworkBehaviour` that negotiates
```

Comment:
While we are at it :)

```suggestion
//! The [`Behaviour`] struct implements a [`NetworkBehaviour`] that negotiates
```